src/components/views/rooms/RoomTile.tsx

```diff
@@ -150,6 +155,8 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
                 CommunityPrototypeStore.getUpdateEventName(this.props.room?.roomId),
                 this.onCommunityUpdate,
             );
+            prevProps.room.off("Room.name", this.onRoomNameUpdate);
```

Comment:
Please use `?.` after `room` on each of these lines, since one or the other `room` maybe be null.

Reply:
Yep good shout @jryans , I've added the optional chaining now :thumbsup: